static/docs/commands-reference/metrics_show.md

```diff
@@ -22,11 +22,18 @@ The optional `path` argument can represent a DVC metric file or a directory. If
 `path` is a directory, recursively search and process all metric files in it
 with the `-R` option.
 
-If metric type is specified (via `-t`) it overrides the type and `--xpath`
-defined (if any) in the corresponding DVC-file (usually, using
-`dvc metrics modify` command). If `-t` is not specified `-x` is ignored. The
-type can be detected by the file extension automatically if the type is
-supported.
+- If metric type (via `-t`) and xpath(`-x`) is specified, it overrides the
```

Comment:
as far as I understand it's enough to specify _only_ `-t` to override the xpath (and make effectively empty if it's not provided explicitly)

Reply:
oh, I see the comment below

Reply:
are specified

Reply:
the override